pandas/tseries/tests/test_timedeltas.py

```diff
@@ -879,10 +879,10 @@ def test_timedelta_ops_with_missing_values(self):
         actual = s1 - timedelta_NaT
         assert_series_equal(actual, sn)
 
-        actual = s1 + NA
```

Comment:
this just proves coercion of ``nan`` -> ``NaT`` when the dtype is appropriate (e.g. ``M8[ns]`` or ``m8[ns]``. 

I would expect this to work (as opposed to raising an error)

```
-> actual = s1 + NA
(Pdb) p s1
0   00:00:01
dtype: timedelta64[ns]
(Pdb) p NA   
nan
(Pdb) p s1 + NA
0   NaT
dtype: timedelta64[ns]
(Pdb) p s1 + pd.NaT
0   NaT
dtype: timedelta64[ns]
```

with dates
```
(Pdb) Series(pd.date_range('20130101',periods=2))        
0   2013-01-01
1   2013-01-02
dtype: datetime64[ns]
(Pdb) Series(pd.date_range('20130101',periods=2))+pd.NaT
*** TypeError: can only operate on a datetimes for subtraction, but the operator [__add__] was passed
(Pdb) Series(pd.date_range('20130101',periods=2))+np.nan
*** TypeError: can only operate on a datetimes for subtraction, but the operator [__add__] was passed
(Pdb) Series(pd.date_range('20130101',periods=2))-np.nan
0   NaT
1   NaT
dtype: timedelta64[ns]
(Pdb) Series(pd.date_range('20130101',periods=2))-pd.NaT
0   NaT
1   NaT
dtype: timedelta64[ns]
```